test/CloudNative.CloudEvents.UnitTests/Kafka/KafkaTest.cs

```diff
@@ -129,6 +129,58 @@ public void KafkaBinaryMessageTest()
             Assert.Equal("value", (string?)receivedCloudEvent["comexampleextension1"]);
         }
 
+        [Theory]
+        [InlineData(MediaTypeNames.Application.Octet, new byte[0])]
+        [InlineData(MediaTypeNames.Application.Json,null)]
+        [InlineData(MediaTypeNames.Application.Xml,"")]
+        [InlineData(MediaTypeNames.Text.Plain,"")]
+        public void KafkaBinaryMessageTombstoneTest(string contentType, object? expectedDecodedResult)
+        {
+            var jsonEventFormatter = new JsonEventFormatter();
+            var cloudEvent = new CloudEvent(Partitioning.AllAttributes)
+            {
+                Type = "com.github.pull.create",
+                Source = new Uri("https://github.com/cloudevents/spec/pull/123"),
+                Id = "A234-1234-1234",
+                Time = new DateTimeOffset(2018, 4, 5, 17, 31, 0, TimeSpan.Zero),
+                DataContentType = contentType,
```

Comment:
While not strictly invalid as far as I'm aware, is there any good reason to have a data content type when there isn't any data?

Reply:
(It looks like you're using that to effectively check the result of decoding different empty bodies - I don't think that's really part of the Kafka code, so I think you can probably simplify this test. If you'd prefer to keep it, that's okay.)

Reply:
I was thinking it's a good idea to verify that the deserialization is well behaved across all the content types, even without a payload.

I can add the null content type as well, to be sure that it works correctly too.

Reply:
But isn't that deserialization purely in the formatter, not in the Kafka code at all?

Reply:
True, it might be a case of over-testing. But I saw it as a better safe than sorry to cover it. 

I can remove the content type checks if you wish?

Reply:
I don't mind too much - it just seems unnecessary. I haven't looked at what the coverage is like at the moment.